src/axom/quest/util/mesh_helpers.cpp

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2017-2023, Lawrence Livermore National Security, LLC and
+// other Axom Project Developers. See the top-level LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+#include "mesh_helpers.hpp"
+
+namespace axom
+{
+namespace quest
+{
+namespace util
+{
+#ifdef AXOM_USE_MFEM
+mfem::Mesh* make_cartesian_mfem_mesh_2D(const primal::BoundingBox<double, 2>& bbox,
```

Comment:
Q: With new MFEM functionality in the code, is there a configuration that builds without MFEM to check  whether AXOM_USE_MFEM is everywhere it needs to be? It looks like all host-configs are using MFEM.

[whitlocb@rzvnc4 host-configs]$ grep MFEM *.cmake | wc -l
21
[whitlocb@rzvnc4 host-configs]$ ls *.cmake | wc -l
21

Reply:
We have several configs without mfem -- see https://github.com/LLNL/axom/blob/develop/scripts/spack/specs.json
* The ones with `+mfem` depend on mfem
* The ones with `~mfem` do not depend on mfem. E.g. 
    * https://github.com/LLNL/axom/blob/develop/scripts/spack/specs.json#L20
    * https://github.com/LLNL/axom/blob/develop/scripts/spack/specs.json#L35-L36

The windows CI also does not depend on mfem

Reply:
Looks like your grep line caught some lines of the form:
https://github.com/LLNL/axom/blob/b233024f2baf616ee99a6a5299781773fa45edbb/host-configs/rzansel-blueos_3_ppc64le_ib_p9-gcc%408.3.1.1.cmake#L80
:)